### PR TITLE
Kleyt0n/issue53

### PR DESCRIPTION
**Pull request by Kleyt0n**

### CODE REVIEW

File: holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py [hunk 1]
```diff
@@ -10,30 +10,97 @@
 
 
 class PluginEstimationAndCalibrationAlgorithm:
+    """Plugin Estimation and Calibration Algorithm
+
+    This class implements the Plugin Estimation and Calibration Algorithm (PECA) for bias mitigation.
+    This algorithm was proposed by Hardt et al. in their paper titled "Fair Regression via Plug-In Estimator and Recalibration".
+    The paper can be found at: https://proceedings.neurips.cc/paper_files/paper/2020/file/ddd808772c035aed516d42ad3559be5f-Paper.pdf
+
+    Parameters
```
Comment (crismunoz) [inline holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py]: Parameters and Attributes are docstrings of __init___ (not the class)
Class must have their description and their reference only that.

File: holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py [hunk 2]
```diff
@@ -10,30 +10,97 @@
...
+        The lambda values used in the calculation of the output predictions.
+
+    References
+    ----------
+    .. [1] Chzhen, E., Denis, C., Hebiri, M., Oneto, L., & Pontil, M. (2020).
+        Fair regression via plug-in estimator and recalibration with statistical guarantees.
+        Advances in Neural Information Processing Systems, 33, 19137-19148.
+    """
+
     def __init__(self, L=25, beta=0.1):
```
Comment (crismunoz) [inline holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py:26]: Definition of L and beta must be here

**Review by crismunoz — CHANGES_REQUESTED** [review]
Please fix the commentaries

File: holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py (hunk 1 above)
Comment (kleyt0n) [reply holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py]: Done.

File: holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py (hunk 2 above)
Comment (kleyt0n) [reply holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py:26]: Done.

File: holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py [hunk 3]
```diff
@@ -10,30 +10,101 @@
...
-        for k in range(len(YT)):
-            v = (
-                self.p[ST[k]] * (YT[k] - i * self.M / self.L) ** 2
-                + (1 - 2 * ST[k]) * self.lambda_
+        transformed_sensitive_features = self.sensitive_groups.transform(
+            sensitive_features, convert_numeric=True
+        )
+        adjusted_predictions = y_pred * 2 - 1
+        index_range = np.arange(-self.length, self.length + 1, 1)
+        output_predictions = np.zeros(len(adjusted_predictions))
```
Comment (crismunoz) [inline holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py]: Does this line make sense?

**Review by crismunoz — DISMISSED** [review]

File: holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py (hunk 3 above)
Comment (kleyt0n) [reply holisticai/bias/mitigation/postprocessing/plugin_estimator_and_recalibration/algorithm.py]: You are right. line removed.

**Review by crismunoz — APPROVED** [review]

**Pull request merged**